site/docs/guide/files.md

```diff
@@ -5,107 +5,112 @@ next: ./games.md
 
 # File Handling
 
-Telegram bots cannot only send and receive text messages, but also all other kinds of messages, such as photos and videos.
+Telegram bots can not only send and receive text messages, but also many other kinds of messages, such as photos and videos.
 This involves handling the files that are attached to the messages.
 
 ## How Files Work for Telegram Bots
 
 > This section explains how files work for Telegram bots.
 > If you want to know how you can work with files in grammY scroll down for [downloading](#receiving-files) and [uploading](#sending-files) files.
 
-Files are stored seperately from messages.
+Files are stored separately from messages.
 A file on the Telegram servers is identified by a `file_id`, which is just a long string of characters.
 
-`AgADBAADZRAxGyhM3FKSE4qKa-RODckQHxsoABDHe0BDC1GzpGACAAEC` is an example for such a `file_id`.
+`AgADBAADZRAxGyhM3FKSE4qKa-RODckQHxsoABDHe0BDC1GzpGACAAEC` is an example of a Telegram `file_id`.
```

Comment:
```suggestion
`AgADBAADZRAxGyhM3FKSE4qKa-RODckQHxsoABDHe0BDC1GzpGACAAEC` is an example of a `file_id`.
```
Never heard the term Telegram `file_id`.

Reply:
Sure, `file_id` is probably clear enough in this case since it's inside of `